app/src/main/java/org/simple/clinic/bloodsugar/entry/BloodSugarEntryEffectHandler.kt

```diff
@@ -7,7 +7,6 @@ import org.simple.clinic.bloodsugar.entry.BloodSugarValidator.Result.ErrorBloodS
 import org.simple.clinic.bloodsugar.entry.BloodSugarValidator.Result.ErrorBloodSugarTooHigh
```

Comment:
Can we reword the commit message to describe what is actually happening?

Ex: "Finish the blood sugar entry activity after the blood sugar is saved"